### PR TITLE
Add 100k open samples

### PR DESCRIPTION
**Pull request by jameshadfield**
See commit messages for details.

While this PR switches the inputs to zst, the outputs are still xz to maintain consistency (xz file sizes are ~27Mb for sequences+metadata).

Docs updated for 100k, but there's still further updates needed for the other files listed on that page.

### CODE REVIEW

File: .github/workflows/rebuild-100k.yml [hunk 1]
```diff
@@ -31,27 +31,60 @@ jobs:
...
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
+
+
     - name: Build info
       run: |
-        echo "--> 100k sample rebuilding on AWS"
+        echo "--> 100k sample rebuilding viw 2 AWS jobs"
```
Comment (huddlej) [inline .github/workflows/rebuild-100k.yml:69]: Should "viw" be "view" here?

File: .github/workflows/rebuild-100k.yml [hunk 2]
```diff
@@ -31,27 +31,60 @@ jobs:
...
+        tail -n1 build-launch-open.log
+        echo
+        JOBID=$( tail -n1 build-launch-gisaid.log | sed -E 's/.+attach ([-a-f0-9]+).+/\1/' )
+        echo "--> View the GISAID job in the AWS console via"
+        echo "    https://console.aws.amazon.com/batch/home?region=us-east-1#jobs/detail/${JOBID}"
         echo
-        JOBID=$( tail -n1 build-launch.log | sed -E 's/.+attach ([-a-f0-9]+).+/\1/' )
-        echo "--> View this job in the AWS console via"
+        JOBID=$( tail -n1 build-launch-open.log | sed -E 's/.+attach ([-a-f0-9]+).+/\1/' )
+        echo "--> View the GISAID job in the AWS console via"
```
Comment (huddlej) [inline .github/workflows/rebuild-100k.yml:88]: I think "the GISAID job" here should be "the open job".

**Review by huddlej — COMMENTED** [review]
Thanks for putting this together, @jameshadfield! I left a couple comments about potential typos, but otherwise the code/config all looks good.

File: .github/workflows/rebuild-100k.yml (hunk 1 above)
Comment (jameshadfield) [reply .github/workflows/rebuild-100k.yml]: It should have been "via" - but I've now edited that line to be more verbose.

File: .github/workflows/rebuild-100k.yml (hunk 2 above)
Comment (jameshadfield) [reply .github/workflows/rebuild-100k.yml]: Yes! Changed. (Although "the open job" has it's own potential for misinterpretation!)

**Comment by jameshadfield** [comment]
CI passes, as did the two (AWS) jobs triggered by [a manual run of the associated GitHub action](https://github.com/nextstrain/ncov/actions/runs/5408786234) so I'm going to merge this

**Pull request merged**